web/src/main/angular/src/app/core/components/transaction-id-search/transaction-id-search-container.component.ts

```diff
@@ -0,0 +1,62 @@
+import {
+    Component,
+    OnInit,
+    OnDestroy,
+    ChangeDetectionStrategy,
+    Output,
+    EventEmitter
+} from '@angular/core';
+
+import {
+    UrlRouteManagerService
+} from 'app/shared/services';
+import {UrlPath} from "../../../shared/models";
+
+@Component({
+    selector: 'pp-transaction-id-search-container',
+    templateUrl: './transaction-id-search-container.component.html',
+    styleUrls: ['./transaction-id-search-container.component.css'],
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class TransactionIdSearchContainerComponent implements OnInit, OnDestroy {
+    @Output() outSearchId: EventEmitter<{txId: string}> = new EventEmitter();
```

Comment:
Declare `@Output` syntax like below for consistency:
```suggestion
    @Output() outSearchId = new EventEmitter<{txId: string}>();
```